<?xml version="1.0"?>

<launch> 

    <node pkg="off_mission" type="off_mission_node" name="off_mission_node" output="screen">
        <rosparam command="load" file="$(find off_mission)/launch/waypoints_xyzy.yaml"/>

    </node>
</launch>
